doc/releases/release-notes-3.7.rst

```diff
@@ -1469,10 +1451,10 @@ Networking
   * Fixed cloning of zero sized network buffers.
   * Added net_buf APIs to handle 40 bit data format.
   * Added receive callback for dummy L2, applicable in certain use cases
-    (for example packet capture).
+    (for example, packet capture).
   * Implemented pseudo interface, a.k.a "any" interface for packet capture use
     case.
-  * Added cooded mode capture support. This allows non-IP based network data capture.
+  * Added coded mode capture support. This allows non-IP based network data capture.
```

Comment:
This should be called "cooked mode", not "coded"

Reply:
Thanks @jukkar. PTAL